pci/src/msi.rs

```diff
@@ -32,7 +37,15 @@ pub fn msi_num_enabled_vectors(msg_ctl: u16) -> usize {
     1 << field
 }
 
-#[derive(Clone, Copy, Default)]
+#[derive(Debug)]
+enum Error {
```

Comment:
Can you also do the `#[derive(Error)]` stuff so we don't have to go back and do it again later.

Reply:
Sure!